source/GameInterface/Services/ItemRosters/Handlers/ItemRosterUpdateHandler.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Common.Messaging;
+using TaleWorlds.CampaignSystem.Party;
+using TaleWorlds.Core;
+using Serilog;
+using Common.Logging;
+using TaleWorlds.CampaignSystem.Settlements;
+using TaleWorlds.CampaignSystem.Roster;
+using GameInterface.Services.ItemRosters.Patches;
+using GameInterface.Services.ItemRosters.Messages;
+using GameInterface.Services.ObjectManager;
+
+namespace GameInterface.Services.ItemRosters.Handlers
+{
+    /// <summary>
+    /// Handles ItemRosterUpdated on client.
+    /// </summary>
+    internal class ItemRosterUpdateHandler : IHandler
+    {
+        private readonly IMessageBroker messageBroker;
+        private readonly ILogger logger;
+        private readonly IObjectManager objectManager;
+
+        public ItemRosterUpdateHandler(IMessageBroker messageBroker, IObjectManager objectManager)
+        {
+            this.messageBroker = messageBroker;
+            this.objectManager = objectManager;
+
+            logger = LogManager.GetLogger<ItemRosterUpdateHandler>();
```

Comment:
Logger should be `private static readonly`
```suggestion
private static readonly ILogger Logger = LogManager.GetLogger<ItemRosterUpdateHandler>();

private readonly IMessageBroker messageBroker;
private readonly IObjectManager objectManager;

public ItemRosterUpdateHandler(IMessageBroker messageBroker, IObjectManager objectManager)
{
    this.messageBroker = messageBroker;
    this.objectManager = objectManager;
```